web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1621,7 +1621,14 @@ fixmystreet.reporting.selectedCategory = function() {
         group_or_cat = $group_or_cat_input.val() || '',
         group_id = group_or_cat.replace(/[^a-z]+/gi, ''),
         $subcategory = $("#subcategory_" + group_id),
-        $subcategory_input = $subcategory.find('input:checked'),
+        // Subcategory may provide a stopper message that disables the report
+        // form while giving the user an external URL to follow. If the user
+        // clicks through this link and then clicks back, they are taken to
+        // the first page of the new report form, but the subcategory will
+        // still be selected behind the scenes (and so the form will still be
+        // disabled). Workaround here is to only pick up on the subcategory
+        // if it is visible (i.e. we are on the subcategory page).
+        $subcategory_input = $subcategory.find(':visible input:checked'),
```

Comment:
I think this can, after looking, be better achieved by, and this is going to sound odd, removing the unload handlers from our OpenLayers library. This means that the FMS page can then be served by Chrome's bf/cache, which fixes the issue entirely because you're taken back to exactly where you were. Didn't realise these handlers were causing this issue, and should make the site better overall too, so this is good!